ansible/roles/debops.tinyproxy/templates/etc/ansible/facts.d/tinyproxy.fact.j2

```diff
@@ -0,0 +1,21 @@
+#!{{ ansible_python['executable'] }}
+
+# {{ ansible_managed }}
+
+from __future__ import print_function
+from json import dumps, loads
+import os
+
+
+def cmd_exists(cmd):
+    return any(
+        os.access(os.path.join(path, cmd), os.X_OK)
+        for path in os.environ["PATH"].split(os.pathsep)
+    )
+
+
+output = loads('''{{ ({
+    "installed": true
```

Comment:
You should set this value dynamically using the `cmd_exists` function. In that case you don't need to wrap this in the `loads()` function, of course.